src/bridge/transactions/kick_off.rs

```diff
@@ -24,6 +26,14 @@ pub struct KickOffTransaction {
     prev_scripts: Vec<Script>,
 }
 
+impl TransactionBase for KickOffTransaction {
+    fn tx(&mut self) -> &mut Transaction { &mut self.tx }
+
+    fn prev_outs(&self) -> &Vec<TxOut> { &self.prev_outs }
+
+    fn prev_scripts(&self) -> Vec<ScriptBuf> { self.prev_scripts.clone() }
+}
+
```

Comment:
The Kick-Off tx is not pre-signed (operator only signs it before sending it to the miners), so I don't think we need it here.